docs/modules/user-guide/partials/proc_specifying-persistent-storage.adoc

```diff
@@ -14,9 +14,9 @@ Components of any type can specify the custom volumes to be mounted on specific
 ----
 schemaVersion: 2.0.0
 metadata:
-  name: MyDevfile
+  name: mydevfile
```

Comment:
`Components of any type can specify the custom volumes to be mounted on specific locations within the image. Note that the volume names are shared across all components`

Not true. Only container component can specify them in devfile v2. Also we may need to say that if a container component is mentioning/mounting it, then there needs be a corresponding volume component of the same name

```yaml
components:
  - name: mydevfile
    container:
      image: golang
      memoryLimit: 512Mi
      mountSources: true
      command: ['sleep', 'infinity']
      volumeMounts:
        - name: cache
          path: /.cache
  - name: cache
    volume:
         size: 2Gi
```